### PR TITLE
Add From field to avm.Send API args

### PR DESCRIPTION
**Pull request by ElectricPanda**
Now when we use avm.Send API, we can choose which address to take asset out with the From field.
If the From field is empty (""), then avm.Send API acts same as before.

### CODE REVIEW

File: vms/avm/service.go [hunk 1]
```diff
@@ -742,6 +744,29 @@ func (service *Service) Send(r *http.Request, args *SendArgs, reply *SendReply)
 		kc.Add(sk)
 	}
 
+	if args.From != "" {
+		fromBytes, err := service.vm.Parse(args.From)
+		if err != nil {
+			return fmt.Errorf("problem parsing to address: %w", err)
```
Comment (danlaine) [inline vms/avm/service.go:750]: I'd change error message to `problem parsing argument 'from' to address`

File: vms/avm/service.go [hunk 2]
```diff
@@ -742,6 +744,29 @@ func (service *Service) Send(r *http.Request, args *SendArgs, reply *SendReply)
...
+		}
+		from, err := ids.ToShortID(fromBytes)
+		if err != nil {
+			return fmt.Errorf("problem parsing to address: %w", err)
```
Comment (danlaine) [inline vms/avm/service.go:754]: I'd change error message to problem parsing argument 'from' to address

File: vms/avm/service.go [hunk 3]
```diff
@@ -683,6 +684,7 @@ type SendArgs struct {
 	Amount   json.Uint64 `json:"amount"`
 	AssetID  string      `json:"assetID"`
 	To       string      `json:"to"`
+	From     string      `json:"from"`
```
Comment (danlaine) [inline vms/avm/service.go:687]: Could you change `from` to be a set of addresses rather than one address?

**Comment by danlaine** [comment]
@ElectricPanda Hey, thanks a lot for making this PR! Much appreciated. I left a few feedback items.

**Comment by ElectricPanda** [comment]
@danlaine Thanks, I fixed error messages and changed args.From to args.Froms!

**Comment by ElectricPanda** [comment]
This PR is replaced with another PR: https://github.com/ava-labs/gecko/pull/376
So I just close this PR.

**Pull request closed without merging**